src/lammps.cpp

```diff
@@ -891,14 +913,23 @@ void LAMMPS::create()
 #endif
 }
 
-/* ----------------------------------------------------------------------
-   check suffix consistency with installed packages
-   invoke package-specific default package commands
-     only invoke if suffix is set and enabled
-     also check if suffix2 is set
-   called from LAMMPS constructor and after clear() command
-     so that package-specific core classes have been instantiated
-------------------------------------------------------------------------- */
+/** Perform LAMMPS class tasks that require all base classes being set up
+ *
+\verbatim embed:rst
```

Comment:
Why are there `rst` commands in the comment? I think this makes the comment block harder to read.

Reply:
This is the only way to insert references to other parts of the manual and write embedded documentation that is consistent with the rest of the manual.

While looking at the docs for the latest version of breathe, I've noticed an alternative tag where the asterisks can remain. Need to check how recent this is and if it works for all existing cases that we already have. https://breathe.readthedocs.io/en/latest/markups.html#handling-leading-asterisks